x-pack/plugins/osquery/server/lib/telemetry/insights/index.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+export { getSessionIDfromKibanaRequest, createAlertStatusPayloads } from './insights';
```

Comment:
All this insights stuff can be nuked in osquery - it needs a lot of work and might not proceed based on the businesses needs.